### ISSUE TITLE
bug: signature with string not properly rendered as of version 1.12.0

### ISSUE THREAD

**Issue by gautierronan** [mkdocstrings/python#191]
Hi, as of version 1.12.0, it seems that our function signatures that include strings are not properly rendered anymore.

For instance, instead of:
```python
wigner(
    state: ArrayLike,
    *,
    ax: Axes | None = None,
    xmax: float = 5.0,
    ymax: float | None = None,
    vmax: float = 2 / jnp.pi,
    npixels: int = 101,
    cmap: str = "dq",                # bug here
    interpolation: str = "bilinear", # and here
    colorbar: bool = True,
    cross: bool = False,
    clear: bool = False
)
```
we have:
```python
wigner(state: ArrayLike, *, ax: Axes | None = None, xmax: float = 5.0, ymax: float | None = None, vmax: float = 2 / jnp.pi, npixels: int = 101, cmap: str = &#39;dq&#39;, interpolation: str = &#39;bilinear&#39;, colorbar: bool = True, cross: bool = False, clear: bool = False)
```
(see the example live on our library documentation, [here](https://www.dynamiqs.org/python_api/plot/wigner.html))

It seems that this bug is somehow related to the use of single quotation marks instead of doubles? I'm not sure whether this is a bug arising from our side, of from the side of `mkdocstrings-python`. Thanks!

**Comment by pawamoy** [comment]
Hi @gautierronan, thanks for the report. It's probably a bug on our end, some HTML that isn't properly escaped. Working on it!

**Comment by pawamoy** [comment]
Could you share your handler configuration? I have trouble reproducing the issue.

**Comment by pawamoy** [comment]
Fixed in v1.12.1 :slightly_smiling_face:

**Comment by gautierronan** [comment]
Thanks @pawamoy for the (very!) quick fix!
